kornia/augmentation/augmentation.py

```diff
@@ -342,3 +342,32 @@ def forward(self, input: UnionType, params: Optional[Dict[str, torch.Tensor]] =
             batch_size: int = self.infer_batch_size(input)
             params = self.get_params(batch_size, height, width, self.degrees, self.translate, self.scale, self.shear)
         return super().forward(input, params)
+
+
+class CenterCrop(AugmentationBase):
+    r"""Crops the given torch.Tensor at the center.
+    Args:
+        size (sequence or int): Desired output size of the crop. If size is an
+            int instead of sequence like (h, w), a square crop (size, size) is
+            made.
+    """
+
+    def __init__(self, size: Union[int, Tuple[int, int]], return_transform: bool = False) -> None:
+        super(CenterCrop, self).__init__(F.apply_center_crop, return_transform)
+        self.size = size
+        self.return_transform = return_transform
+
+    @staticmethod
+    def get_params(size: Union[int, Tuple[int, int]]) -> Dict[str, torch.Tensor]:
+        if isinstance(size, tuple):
+            size_param = torch.tensor([size[0], size[1]])
+        elif isinstance(size, int):
+            size_param = torch.tensor([size, size])
+        else:
+            raise Exception(f"Invalid type: {type(size)}")
```

Comment:
add what the expected type is. We also might want to accept size to be a tensor, just for consistency, but I think this can be discussed in a separte PR after release

Reply:
yes, I was thinking the same, but it might imply some changes inside `center_crop`. We need to discuss the functionality - being a tensor means that a network could predict this values, however it could make impossible to batch this operator unless we return a Llist of tensors

Reply:
Lets keep it like that for now. We can discuss this further in the future if the necessity arises.